build/dependency

```diff
@@ -78,49 +132,40 @@ if [ -z "$pkg_prefix" ]; then
 fi
 export pkg_prefix
 cd $pkg_rootdir
-if [ -x build/spec/$1 ]; then
-    exec build/spec/$1  # If the spec is executable, pass control to it
-fi
```

Comment:
Now all the software we build follows the spec. No need for this shortcut anymore.